### DISCUSSION TITLE
Alternative text - Accessibility

### DISCUSSION THREAD

**Discussion by neiltomlin** [plouc/nivo#1587] [Ideas]
https://www.w3.org/TR/WCAG21/#non-text-content
https://dequeuniversity.com/rules/axe/4.1/svg-img-alt

The rendered SVG for the Pie/Donut has no title attribute or child text element that describes the graphic. This is required by widely accepted web accessibility standards.

Can the component be augmented to take a Prop to pass any or all of the following?

- Text to a title attribute or element
- Text to an aria-label attribute
- An ID to an aria-labelledby attribute

**Comment by wyze** [comment]
I agree we need to be more accessible with all the charts. It looks like a couple of your suggestions were mentioned in https://github.com/plouc/nivo/issues/126 already. We just have not got around to implementing them yet.
